config/csi/kustomization.yaml

```diff
@@ -2,5 +2,5 @@
 namespace: ibm-spectrum-scale-csi
 
 resources:
-- github.com/IBM/ibm-spectrum-scale-csi.git//operator/config/overlays/cnsa?ref=release-2.11
+- github.com/IBM/ibm-spectrum-scale-csi.git//operator/config/overlays/cnsa?ref=v2.11.0
```

Comment:
nice, yea... we should revisit this on GA and lock to a tag rather than a branch.